packages/live-update/Package.swift

```diff
@@ -12,7 +12,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", branch: "6.0.0"),
         .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.9.0")),
-        .package(url: "https://github.com/ZipArchive/ZipArchive.git", .upToNextMajor(from: "2.2.3"))
+        .package(url: "https://github.com/ZipArchive/ZipArchive.git", exact: "2.2.3")
```

Comment:
The downside is that every other plugin that uses `ZipArchive` must now also use exactly this version. I'm not that familiar with SPM yet. Is there no way that you can specify the version you want to use in your project as long as it's compatible with the plugins version?

Reply:
latest version 2.5.x have minimum ios version ios 15.5 so it doesnt work.
Check my discussion here
https://github.com/capawesome-team/capacitor-plugins/discussions/386

Reply:
Using exact version for 2.4.3 works good in my project now. What do you mean all other plugin that uses  ZipArchive?

Reply:
Yes, but other users may have other plugins that use this dependency. Then you get version conflicts. That's why I want to leave the version as free as possible. But you have convinced me that it is not a solution to increase the minimum deployment target.

Reply:
yeah true. The best solution is not to have such library and just have some pure swift code for this without many dependencies.

Reply:
Yes, that's right. If you ever get bored, feel free to create a PR. ^^ I would love to throw out dependencies as long as the new code is maintainable.